### DISCUSSION TITLE
Account for path in APP_URL

### DISCUSSION THREAD

**Discussion by gbkwiatt** [envault/envault#62] [Ideas]
Just a suggestion, I have it deployed with alias on my server  like `server.com/envault`  and defined `APP_URL=server.com/envault`
But nowhere in app it accounts for `parse_url(config(app_url))['path']`  so I had to add it manually in couple of places.

Either check path, or create extra .env APP_PATH and use that instead ?

**Comment by danharrin** [comment]
Hosting it on a subdirectory is not exactly "supported" behaviour at the moment so I'm not sure we should accommodate those changes yet.
